index.html

```diff
@@ -98,6 +98,19 @@
 
     <div class="content">
       <main class="container">
+
+        <div class="messages__wrapper layout-container grid-full">
+          <div role="contentinfo" aria-label="Success message" class="messages messages--success">
+            <img src="images/success.svg" alt="success">
+            <div class="alert" role="alert">
+              <h2 class="visually-hidden">Success message</h2>
+              Get involved.
+              <span class="supporting-text">Looking to contribute? Follow this <a href="https://www.drupal.org/docs/8/themes/olivero/how-to-contribute-to-olivero">link</a> to see how.</span>
+            </div>
+            <img src="images/close.svg" alt="close" class="close">
```

Comment:
The close button needs to be a `<button>` element. Buttons respect click links and have more semantic meaning than a standard img.

We also should use CSS pseudo-elements for the X instead of an IMG. This will allow us to easily change the colors of the lines on hover/focus events.

Reply:
Once we get the button element in, we also need to attach JS to it. I can help with that if needed